### PR TITLE
 Use Redux in React components

### PR DESCRIPTION
**Pull request by AbdifatahYasin1**
###  Use Redux in React components.

In this step you 
- [ ] I used Redux store to display books.
- [ ] I also added some UI elements to your components and make them dispatch actions.

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  };
+
+  return (
+
+    <div className="book">
+
+      <h2 className="title">{title}</h2>
+      <h6 className="author">{author}</h6>
+
+      <button className="btn-remove" type="button" onClick={handleRemove}>
```
Comment (muneebulrehman) [inline src/components/Books.js:20]: Removing a book is not working as expected. Kindly have a look at [this video](https://www.loom.com/share/157ac51d2b664b7cb0dd583bd7006480) I recorded where I demonstrated what exactly is going wrong. Kindly fix this issue and then submit for a re-review.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @AbdifatahYasin1 , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Good docs

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 1 above)
Comment (AbdifatahYasin1) [reply src/components/Books.js:20]: @muneebulrehman thanks my bro you are the only reviewer who recorded a video and helped me a lot.

Comment (BenMukebo) [reply src/components/Books.js:20]: Great job @muneebulrehman for the video👏🏻 🥇

File: src/components/Form.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  const dispatch = useDispatch();
+  const [book, setBook] = useState({ title: '', author: '' });
+
+  const handleChange = (event) => {
+    setBook({ ...book, [event.target.name]: event.target.value });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const newBook = { id: Date.now(), ...book };
```
Comment (BenMukebo) [inline src/components/Form.js:15]: - [OPTIONAL] I can suggest you use `uuidv4` library for the ids generator this is better than using `Date.now()`  for more documentation you can visit this link [uuidv4](https://www.npmjs.com/package/uuidv4) 👍🏻

**Review by BenMukebo — APPROVED** [review]
### Hi @AbdifatahYasin1, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL ]  I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```

- [OPTIONAL ] When I click the link to the `MIT.md` file that you added, I get this 404 `error`. Kindly  update this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional 😉 👍🏻 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**